Tests/Functional/SelectedTest/main.swift

```diff
@@ -26,19 +26,20 @@ class ExecutedTestCase: XCTestCase {
 // CHECK-TESTCASE: Test Case 'ExecutedTestCase.test_bar' passed \(\d+\.\d+ seconds\).
 // CHECK-ALL:      Test Case 'ExecutedTestCase.test_bar' started.
 // CHECK-ALL:      Test Case 'ExecutedTestCase.test_bar' passed \(\d+\.\d+ seconds\).
-    func test_bar() {}
+    func test_bar() {
+    }
```

Comment:
Here again I personally prefer the former syntax. Why use two lines to represent an empty function? Is the value added here worth making this change?